pkg/kube/pod_controller.go

```diff
@@ -204,12 +211,24 @@ func (p *podController) StopPod(ctx context.Context, stopTimeout time.Duration,
 	return nil
 }
 
+// getContainerName returns container name, which should be passed to operations which requires it
+// if container name was specified in podOptions, it will be used.
+// Otherwise, the first container name from specs will be taken as best effort (when pods are created with sidecars
+// sidecar containers are placed after main container).
```

Comment:
```suggestion
// getContainerName returns container name, which should be passed to
// operations that require it.
// If the container name was specified in podOptions, it will be used.
// Otherwise, the first container name from specs will be taken as best effort
// (when pods are created with sidecars, sidecar containers are placed after
// main container).
```
80 char wrap